src/components/form/ErrorLabel.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './ErrorLabel.css'
+
+const ErrorLabel = ({ errorMessage }) => (
+  <div className="FormEdit">
+    <p>
+      <i className="fas fa-exclamation-triangle" aria-hidden="true" />&nbsp;
+      <span>{errorMessage}</span>
+    </p>
+    <p onClick={() => {window.location.reload()}}> Click here to reload the page </p>
```

Comment:
Creating a new anonymous function here every render seems unnecessary. Why not just `onClick={window.location.reload}`?

Reply:
Try this:

```javascript
f = window.location.reload
f()
```
You will get error:

```
VM54:1 Uncaught TypeError: Illegal invocation
    at <anonymous>:1:1
(anonymous) @ VM54:1
```

So, basically, window.location.reload must be called as a method of the location object, and bound to location. It would be sufficient to break the anonymous function out into a named function so that a new anonymous function is not declared on each render.

Reply:
Fixed in 91a8dbb